php5.6-fpm/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN DEBIAN_FRONTEND=noninteractive \
     mysql-client \
     unzip \
     git \
+    imagemagick \
```

Comment:
@cableman, why is this added to the PR called “Fixed completion and ":"”?

Reply:
Because my git-voodoo failed here